### ISSUE TITLE
Colorization changed/broken in master

### ISSUE THREAD

**Issue by dur-randir** [garu/Data-Printer#123]
It's too lengthy to describe, just compare old/new output for 

```
perl -e 'use DDP; $z="123"; $y=bless \$z; p $y'
```

some parts have lost color, some parts have changed colors.

**Comment by garu** [comment]
@dur-randir Yes! I have improved the amount of colorization possibilities (fixes #8) and added support for themes! They are still incomplete except for two: `Material` (the new default) and `Classic` (the old default). You may now create your own themes (should fix #43) and upload them to CPAN or use them locally in your project. It's also much simpler to switch between themes. Instead of updating each color, just: `use DDP theme => 'Classic'` (or any other).

On a related note: **thank you** for taking the time to help me review Data::Printer. I'm sorry I ended up with a single commit and not documenting anything. Most of the work was during the PTS hackathon while I was refactoring the internals to provide an object (fixes: #46) so I didn't have a lot of time to properly separate things but still felt it was better to have more people exposed to the new code quickly than take several weeks releasing them neatly. **Your review is a huge help** to understand what was actually fixed and what is still (or suddenly became) broken.

**Comment by dur-randir** [comment]
>use DDP theme => 'Classic'

It didn't quite work out:

```
% perl -Iblib/lib -Iblib/arch  -e 'use DDP; $z="123"; $y=bless \$z; p $y'
Use of uninitialized value in join or string at blib/lib/Data/Printer/Theme.pm line 143.
Use of uninitialized value in join or string at blib/lib/Data/Printer/Theme.pm line 143.
Use of uninitialized value in join or string at blib/lib/Data/Printer/Theme.pm line 143.
Use of uninitialized value in join or string at blib/lib/Data/Printer/Theme.pm line 143.
Use of uninitialized value in join or string at blib/lib/Data/Printer/Theme.pm line 143.
Use of uninitialized value in join or string at blib/lib/Data/Printer/Theme.pm line 143.
Use of uninitialized value in join or string at blib/lib/Data/Printer/Theme.pm line 143.
Use of uninitialized value in join or string at blib/lib/Data/Printer/Theme.pm line 143.
Use of uninitialized value in join or string at blib/lib/Data/Printer/Theme.pm line 143.
Use of uninitialized value in join or string at blib/lib/Data/Printer/Theme.pm line 143.
Use of uninitialized value in join or string at blib/lib/Data/Printer/Theme.pm line 143.
Use of uninitialized value in join or string at blib/lib/Data/Printer/Theme.pm line 143.
Use of uninitialized value in join or string at blib/lib/Data/Printer/Theme.pm line 143.
Use of uninitialized value in join or string at blib/lib/Data/Printer/Theme.pm line 143.
Use of uninitialized value in join or string at blib/lib/Data/Printer/Theme.pm line 143.
Use of uninitialized value in join or string at blib/lib/Data/Printer/Theme.pm line 143.
Use of uninitialized value in join or string at blib/lib/Data/Printer/Theme.pm line 143.
ain  {
    internals:
}
```

- first symbol of 'main' is lost
- internals are totally lost

**Comment by dur-randir** [comment]
>Most of the work was during the PTS hackathon while I was refactoring the internals to provide an object (fixes: #46) so I didn't have a lot of time to properly separate things but still felt it was better to have more people exposed to the new code quickly than take several weeks releasing them neatly.

Changelog/deprecations is going to be huge :)
